packages/data-store/src/entities/contact/ElectronicAddressEntity.ts

```diff
@@ -32,6 +32,9 @@ export class ElectronicAddressEntity extends BaseEntity {
   })
   party!: PartyEntity
 
+  @Column({ name: 'partyId', nullable: true })
+  partyId!: string
```

Comment:
The collumn is nullable, but the partyId has an !; is that supposed to be?

Reply:
fixed